novawallet/Modules/Swaps/Setup/SwapSetupPresenter.swift

```diff
@@ -134,21 +134,29 @@ final class SwapSetupPresenter {
         view?.didReceiveAmount(receiveInputViewModel: amountInputViewModel)
     }
 
-    private func absoluteValue(for input: AmountInputResult?) -> Decimal? {
+    private func getPayAmount(for input: AmountInputResult?) -> Decimal? {
         guard
             let input = input,
             let payChainAsset = payChainAsset else {
             return nil
         }
-        guard let transferrableBalanceDecimal =
-            Decimal.fromSubstrateAmount(
-                assetBalance?.transferable ?? 0,
-                precision: payChainAsset.asset.displayInfo.assetPrecision
-            ) else {
-            return nil
+        let includedFee: BigUInt
+        switch input {
```

Comment:
We don't need this switch case. Because in case of absolute value the input will use it's own internal value.